runelite-client/src/main/java/net/runelite/client/plugins/wiki/WikiPlugin.java

```diff
@@ -129,6 +142,16 @@ public void shutDown()
 		});
 	}
 
+	@Subscribe
+	public void onConfigChanged(ConfigChanged event)
+	{
+		if (event.getGroup().equals("wiki"))
+		{
+			shutDown();
```

Comment:
Don't invoke shutDown and startUp methods directly. Move the code called from there into other methods and call those.